R/validate.R

```diff
@@ -37,6 +38,12 @@ validate_count_mat <- function(count_mat) {
   if (any(is.infinite(count_mat@x))) {
     return(err("matrix values must not be infinite"))
   }
+  if (!all(sapply(barcodes, nzchar))) {
+    return(err("barcodes cannot be the empty string"))
```

Comment:
is this being added as part of solving the issue, or is it some missing validation that was caught + updated unrelated to the change?